test/testdata/test-cases/array.h

```diff
@@ -0,0 +1,35 @@
+#include <stdint.h>
+
+
+/*
```

Comment:
I think you can remove this comment. If something is missing, we will get hold of it eventually.